docs/rules/block-attributes-order.md

```diff
@@ -0,0 +1,129 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/block-attributes-order
+description: enforce order of block attributes
+since: v4.3.0
```

Comment:
This rule is not yet released. Please remove this line and re-run `npm run update`.